src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,79 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final String COMMA = ",";
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result";
+    private int supplyAmount = 0;
+    private int buyAmount = 0;
```

Comment:
These class fields should be local variables in the method where they are used since they don't represent the state of the class but are rather used for calculations within a particular operation. Refactor `supplyAmount` and `buyAmount` to be local variables in the `readFromFile` method.